weave/trace/op.py

```diff
@@ -232,6 +243,7 @@ def finish(output: Any = None, exception: Optional[BaseException] = None) -> Non
             output,
             exception,
             postprocess_output=__op.postprocess_output,
+            op=__op,
```

Comment:
Can probably get rid of the `postprocess_output` if we're passing the entire `__op`

Reply:
why do you need to pass the whole `op`?

Reply:
Just seemed a little cleaner to pass in the Op object rather than to have separate arguments for the handlers.

Reply:
ok I would pick one -- if you want to just pass the `op` that's fine, but then we should take out `postprocess_output` for consistency